### PR TITLE
[feat.] add autoupdate

### PR DESCRIPTION
**Pull request by xyloflake**
# Auto Updates with electron-builder
- Tested with @gigabytebit

> [!IMPORTANT]
> Autoupdate will work only for windows and linux

### CODE REVIEW

**Review by gamersi — DISMISSED** [review]
LGTM

**Comment by staniel359** [comment]
@xyloflake Is it with auto-downloading?

**Comment by xyloflake** [comment]
> @xyloflake Is it with auto-downloading?

@staniel359 Yes, it downloads automatically

File: electron/actions/app/checkForUpdates.js [hunk 1]
```diff
@@ -93,14 +99,19 @@ function handleError () {
...
-    'https://api.github.com/repos/staniel359/muffon/releases/latest'
-
-  net.fetch(
-    releasesUrl
-  ).then(
-    handleSuccess
-  ).catch(
-    handleError
-  )
+  if(isMac) {
```
Comment (staniel359) [inline electron/actions/app/checkForUpdates.js]: @xyloflake Please add a space after `if`.

Comment (xyloflake) [reply electron/actions/app/checkForUpdates.js]: Okay

**Review by staniel359 — APPROVED** [review]

**Review by gamersi — APPROVED** [review]
LGTM

**Pull request merged**